app/helpers/fhir_helper.rb

```diff
@@ -230,10 +268,13 @@ def to_us_core_ethnicity(ethnicity)
 
   # Return a string representing the ethnicity of the given FHIR::Patient
   def from_us_core_ethnicity(patient)
-    url = 'us-core-ethnicity'
-    code = patient&.extension&.select { |e| e.url.include?(url) }&.first&.extension&.select { |e| e.url == 'ombCategory' }&.first&.valueCoding&.code
+    urls = %w[ombCategory http://hl7.org/fhir/StructureDefinition/data-absent-reason]
+    ethnicity = patient&.extension&.select { |e| e.url.include?('us-core-ethnicity') }&.first&.extension&.select { |e| urls.include?(e.url) }&.first
+    code = ethnicity&.valueCoding&.code || ethnicity&.valueCode
     return 'Hispanic or Latino' if code == '2135-2'
     return 'Not Hispanic or Latino' if code == '2186-5'
+    return 'Unknown' if code == 'unknown'
+    return 'Refused to Answer' if code == 'asked-declined'
```

Comment:
Can't comment on the line directly, but right now the new ethnicity values are not updated in this line for the API: https://github.com/SaraAlert/SaraAlert/blob/07ea552442160d2386e34d5ae0cfcdb676fa38de/app/helpers/fhir_helper.rb#L210

I feel like this line should be pulling from the values in ValidationHelper also so it's one less place we have to update in the future.

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/5bf58817fd305885bab670ab491e78dea615eeaf#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6R242

> > 2. In the "Guidance" tab of the updated Sara Alert Import Format file, you'll need to update the possible values for "Ethnicity".
> 
> @timwongj Could you also update the date in the "Updates" tab as well to be 3/9? Also annoying thing about excel, but it opens on the tab and with the selection you had when you saved it, so could you also select the the leftmost cell of each tab and save on the "Updates" tab?

fixed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/f87317c97428ade207f5630c951f2c0a9dd827b8